include/HttpRequestParser.hpp

```diff
@@ -5,20 +5,22 @@
 #include <string>
 #include <iostream>
 #include <sstream>
+#include <map>
 
 class HttpRequestParser
 {
 	public:
 		HttpRequestParser();
 		~HttpRequestParser();
 
-		HttpRequest					parseHttpRequest(std::string request);
-		void						parseRequestLine(std::string &requestLine, std::string &method, std::string &uri, std::string &version);
-		std::string					parseMethod(std::string &requestLine);
-		const std::string			parseVersion(std::string &requestLine);
-		const std::string			parseUri(std::string &requestLine);
-		const std::string			parseHost();
-		const std::string			parseBody();
-		int							parseContentLength();
-		int							compareMethod(std::string method, std::string &requestLine);
+		HttpRequest				parseHttpRequest(std::string request);
```

Comment:
these should be private if they are used only inside the HttpRequestParser itself

Reply:
And yes, forgot to change that. Good catch.